robottelo/ui/locators.py

```diff
@@ -753,6 +757,36 @@ def __iter__(self):
 
 locators = LocatorDict({
 
+    # Bookmarks
+    "bookmark.select_name": (
+        By.XPATH,
+        ("//td[following-sibling::td[text()='%s']]"
+         "/a[contains(@href,'bookmarks')][span[contains(.,'%s')]]")),
```

Comment:
Missed your second message. If you're talking about `/a[smth][smth]` - it's `and`.